lib/signaling/v3/remoteparticipant.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+const RemoteParticipantV2 = require('../v2/remoteparticipant');
```

Comment:
Was there something blocking new code to be written in typescript? was it because of the existing `js` requires?

Reply:
I want the typescript conversion to happen top-down, with public classes being converted first. I don't want to be inn a situation where some public classes are in ts, some are in js, some internal classes are in ts, some are in js.

Reply:
This is definitely a discussion for the JS team to have, but I think the time to have it is before tickets are cut and code is written. I think we missed the boat to transition internal classes to .ts files this time.